FreshRSS/FreshRSS
# Improved: config dropdown structure

The config dropdown menu has now a better structure that represents the true semantic tree: "headlines" and its "sub-trees".

(Another PRs will follow to improve the menu semantic structure of aside navigations)

The layout of the menu is a little bit improved (use `rem` values instead of `px`, that gives a bit more empty space around the text)

Changes proposed in this pull request:

- phtml
- (S)CSS


How to test the feature manually:

1. test all themes
2. open the config dropdown menu
3. check it in mobile view and in normal view

Pull request checklist:
